IdentityCore/src/requests/MSIDBrokerTokenRequest.m

```diff
@@ -174,7 +177,8 @@ - (NSDictionary *)defaultResumeDictionaryContents
       @"redirect_uri"     : self.requestParameters.redirectUri,
       @"correlation_id"   : self.requestParameters.correlationId.UUIDString,
 #if TARGET_OS_IPHONE
-      @"keychain_group"   : self.requestParameters.keychainAccessGroup ?: MSIDKeychainTokenCache.defaultKeychainGroup
+      @"keychain_group"   : self.requestParameters.keychainAccessGroup ?: MSIDKeychainTokenCache.defaultKeychainGroup,
+      @"broker_nonce"     : self.brokerNonce
```

Comment:
nit: can broker nonce be nil?

Reply:
To make sure it won't be nil, I added a getter to always init the nonce if it is not set yet.